src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -3,6 +3,7 @@
   (:require [re-frame.core :as re-frame]
             [reagent.core :as reagent]
             [status-im.ui.components.react :as react]
+            [taoensso.timbre :as log]
```

Comment:
unused

Reply:
@yenda  hey how do you even see that from github review screen?

Reply:
I infered it because if it was not there before it wouldn't have compiled since it is the only change to the file.